mvlearn/utils/utils.py

```diff
@@ -53,17 +77,19 @@ def check_Xs(Xs, multiview=False, enforce_views=None, copy=False):
         else:
             Xs = list(Xs)
 
-    if len(Xs) == 0:
+    n_views = len(Xs)
+    if n_views == 0:
         msg = "Length of input list must be greater than 0"
         raise ValueError(msg)
 
     if multiview:
-        if len(Xs) == 1:
+        if n_views == 1:
             msg = "Must provide at least two data matrices"
             raise ValueError(msg)
-        if enforce_views is not None and len(Xs) != enforce_views:
-            msg = "Wrong number of views. Expected {} " \
-                " but found {}".format(enforce_views, len(Xs))
+        if enforce_views is not None and n_views != enforce_views:
```

Comment:
Just a comment but we are supporting only Python 3 so using f strings is fine if you want.

Reply:
sure. I ran black on my code once I wrote it, and it reformatted some things.